pgbot/__init__.py

```diff
@@ -173,7 +173,7 @@ async def message_delete(msg: discord.Message):
         for log in common.cmd_logs.keys():
             if common.cmd_logs[log].id is not None:
                 if common.cmd_logs[log].id == msg.id:
-                    del common.cmd_logs[log]
+                    common.cmd_logs[log] = None
```

Comment:
Why this change? If it is like this, won't the dict fill up with `None` for deleted commands rather than the dict size reducing?